clients/python/test/e2e/test_notebooks.py

```diff
@@ -66,6 +66,6 @@ def test_run_notebooks(tmp_path: Path, notebook: Path, params: dict[str, Any]):
         input_path=tmp_nb,
         output_path=output,
         kernel_name="python3",
-        parameters=params or {},
+        parameters={}, # NOTE: these notebooks for the moments have not parameters
```

Comment:
The reason for the `params` input is that this function is also used simply as a function where it is called in line 30 above. I.e. with this change `test_notebook_config` will fail. The way I use the `params` is to ensure my jupyter environment/settings are setup correctly when using papermill.

Reply:
aha, I see now. 
In my experience that it is a bit _unusual_ to explicitly call a test :-) That is why i missed it.

I would rather create a separate helper. Note that test dependencies change quickly and therefore having an extra helper makes it more flexible.